### PR TITLE
wrapper for get_code.tdata

### PR DESCRIPTION
**Pull request by mhallal1**
related to https://github.com/insightsengineering/teal.code/pull/79

### CODE REVIEW

File: R/tdata.R [hunk 1]
```diff
@@ -94,12 +94,22 @@ get_code.tdata <- function(data) {
...
 
+#' Wrapper for `get_code.tdata`
+#' This wrapper is to be used by downstream packages to extract the code of a `tdata` object
+#'
+#' @param data (`tdata`) object
+#'
+#' @return (`character`) code used in the `tdata` object.
+#' @export
+get_code_tdata <- function(data) {
+  get_code.tdata(data)
```
Comment (nikolas-burkoff) [inline R/tdata.R:105]: So I think this should be:

```suggestion
  checkmate::assert_class(data, "tdata")
  get_code(data)
```

**Review by nikolas-burkoff — APPROVED** [review]
1 small comment then this can be merged in - I've change branch to teal_refactor@main


I guess we can open PRs in the modules to use this and also make changes for warnings

**Review by nikolas-burkoff — APPROVED** [review]
I'd add a couple of tests too

**Comment by mhallal1** [comment]
@nikolas-burkoff tests added for this functionality.
I also created an issue to test `tdata` which is not tested at all: https://github.com/insightsengineering/teal/issues/741

**Review by gogonzo — APPROVED** [review]

**Pull request merged**